.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ repos:
         args: ["--allow-missing-credentials"]
 
   - repo: https://github.com/pre-commit/pygrep-hooks
-    rev: v1.9.0
+    rev: v1.10.0
```

Comment:
this PR should only contain your changes. why are other PR changes coming into this. Please rebase and force-push